### PR TITLE
feat: send ui integration

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: packages/coin-support-btc/src/index.ts [hunk 1]
```diff
@@ -79,4 +82,12 @@ export class BtcSupport implements CoinSupport {
...
     throw new Error(`Method not implemented`);
   }
+
+  public getTransactionLink(txn: ITransaction): string {
+    return getTransactionLink(
+      coinList[txn.assetId] as IBtcCoinInfo,
+      txn.hash,
+      !!txn.confirmations,
+    );
+  }
```
Comment (irshadCypherock) [inline packages/coin-support-btc/src/index.ts]: This already exists on `history` PR as `getExplorerLink`

File: packages/cysync-core/src/components/GenericLoader.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { DialogBox, Image, loaderGrayIcon } from '@cypherock/cysync-ui';
+import React from 'react';
+
+export const GenericLoader: React.FC = () => (
```
Comment (irshadCypherock) [inline packages/cysync-core/src/components/GenericLoader.tsx]:
```suggestion
export const LoaderDialog: React.FC = () => (
```

File: packages/cysync-core/src/dialogs/Send/Dialogs/Components/AddressAndAmountSection/AmountInput.tsx [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+  initialValue,
+  onChange,
+  converter,
+}) => {
+  const throbber: JSX.Element = <Throbber size={15} strokeWidth={2} />;
+  const [isLoading, setIsLoading] = useState(false);
+  const [firstValue, setFirstValue] = useState<string>(initialValue ?? '');
+  const [secondValue, setSecondValue] = useState<string>(
+    converter(initialValue ?? '0') || '',
+  );
```
Comment (irshadCypherock) [inline packages/cysync-core/src/dialogs/Send/Dialogs/Components/AddressAndAmountSection/AmountInput.tsx]: Can we rename these variables?

File: packages/cysync-core/src/dialogs/Send/Dialogs/Components/FeeSection/BitcoinInput.tsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  initialValue: number;
+  isTextInput: boolean;
+  unit: string;
+  onChange: (newValue: number) => void;
+}
+const captions = [
+  { id: 1, name: 'Min' },
+  { id: 2, name: 'Average' },
+  { id: 3, name: 'Max' },
+];
```
Comment (irshadCypherock) [inline packages/cysync-core/src/dialogs/Send/Dialogs/Components/FeeSection/BitcoinInput.tsx]: Text should be from i18n

File: packages/cysync-core/src/dialogs/Send/Dialogs/Components/FeeSection/FeesHeader.tsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  Typography,
+  ButtonAttributes,
+  PillButtonToggle,
+} from '@cypherock/cysync-ui';
+import React from 'react';
+
+const Buttons: ButtonAttributes[] = [
+  { id: 1, label: 'Standard', type: 'slider' },
+  { id: 2, label: 'Advanced', type: 'input' },
+];
```
Comment (irshadCypherock) [inline packages/cysync-core/src/dialogs/Send/Dialogs/Components/FeeSection/FeesHeader.tsx]: Text should be from i18n

File: packages/cysync-core/src/dialogs/Send/Dialogs/FinalMessage.tsx [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+                  <Typography variant="span" color="muted" $fontSize={14}>
+                    <LangDisplay text={displayText.hashLabel} />
+                  </Typography>
+                </Flex>
+                <Flex align="center" direction="row" gap={8}>
+                  <Button
+                    variant="none"
+                    onClick={() => {
+                      // TODO: open external link
+                      window.open(transactionLink);
```
Comment (irshadCypherock) [inline packages/cysync-core/src/dialogs/Send/Dialogs/FinalMessage.tsx]: Is this todo done?

File: packages/cysync-core/src/dialogs/Send/context/index.tsx [hunk 7]
```diff
@@ -60,50 +95,315 @@ export const SendDialogProvider: FC<SendDialogContextProviderProps> = ({
 }) => {
   const lang = useAppSelector(selectLanguage);
   const dispatch = useAppDispatch();
+  const { priceInfos } = useAppSelector(selectPriceInfos);
   const deviceRequiredDialogsMap: Record<number, number[] | undefined> = {
-    1: [0],
+    3: [0],
+    4: [0],
+    5: [0],
   };
```
Comment (irshadCypherock) [inline packages/cysync-core/src/dialogs/Send/context/index.tsx]: I think we only require device connection at 3rd tab, right?

File: packages/cysync-core/src/dialogs/Send/context/index.tsx [hunk 8]
```diff
@@ -60,50 +95,315 @@ export const SendDialogProvider: FC<SendDialogContextProviderProps> = ({
...
+  useEffect(() => {
+    resetInputStates();
+  }, [selectedAccount, selectedWallet]);
+
+  const resetStates = () => {
+    setSignedTransaction(undefined);
+    setTransactionHash(undefined);
+    setTransactionLink(undefined);
+    setError(undefined);
+  };
```
Comment (irshadCypherock) [inline packages/cysync-core/src/dialogs/Send/context/index.tsx]: Clear the device states as well

**Review by irshadCypherock — CHANGES_REQUESTED** [review]

File: packages/cysync-core/src/components/GenericLoader.tsx (hunk 2 above)
Comment (TejasvCypherock) [reply packages/cysync-core/src/components/GenericLoader.tsx:4]: Let's do this rename in a separate PR, as this is used by many other dialogs

File: packages/cysync-core/src/dialogs/Send/context/index.tsx (hunk 7 above)
Comment (ghost) [reply packages/cysync-core/src/dialogs/Send/context/index.tsx]: We require 3rd and 4th, updated.

File: packages/coin-support-btc/src/index.ts (hunk 1 above)
Comment (ghost) [reply packages/coin-support-btc/src/index.ts]: Merged history branch in this one

**Review by irshadCypherock — APPROVED** [review]

**Pull request merged**